### PR TITLE
Fix overflowing labels and values in dropdowns

### LINKED ISSUE #573: Dropdown text appears below arrow
image

### PR DESCRIPTION
**Pull request by ScopeyNZ**
Fixes:
- Dropdown labels appear under the dropdown arrow
- Dropdown values overflow the dropdown
- An issue where labels overlayed the selected value

Added:
- The ability to specify whether you want the dropdown to expand on the y-axis

Also:
- Put some stuff in alphabetical order

Fixes #573

### CODE REVIEW

**Review by kvisca — APPROVED** [review]
LGTM

File: src/stories/components/ChecDropdown.stories.mdx [hunk 1]
```diff
@@ -371,3 +371,62 @@ Todo list
     }}
   </Story>
 </Preview>
+
+# With long labels
```
Comment (robbieaverill) [inline src/stories/components/ChecDropdown.stories.mdx:375]: These shouldn't be h1s, minor point

**Comment by robbieaverill** [comment]
image

Did you want to break the values as well?

File: src/stories/components/ChecDropdown.stories.mdx (hunk 1 above)
Comment (ScopeyNZ) [reply src/stories/components/ChecDropdown.stories.mdx:375]: Yeah I thought this, but I was just copying the other titles.

**Comment by ScopeyNZ** [comment]
I suppose it would look neater to put an ellipsis on those too. I didn't touch the actual dropdown, but it should be an easy addition. I'll do it

**Comment by ScopeyNZ** [comment]
Updated.

**Pull request merged**